app/controllers/events_controller.rb

```diff
@@ -1,6 +1,11 @@
 class EventsController < ApplicationController
   def index
     @events = Event.upcoming
+    if (@keyword = params["search"]).present?
+      keywords = @keyword.downcase.split.join("|")
+      where_clause = "lower(name) similar to '%(#{keywords})%'"
+      @events = @events.where(where_clause)
```

Comment:
We prefer to be expressive in Ruby. So get rid of `where_clause`. Keep it one line.